packages/forms/src/UIForm/UIForm.component.js

```diff
@@ -18,6 +18,10 @@ import { I18N_DOMAIN_FORMS } from '../constants';
 import '../translate';
 import theme from './UIForm.scss';
 
+export const INVALID_ATTRIBUTE = 'aria-invalid';
+
+const INVALID_SELECTOR = `[${INVALID_ATTRIBUTE}="true"]`;
```

Comment:
This feels like `const selector = 'div'` and use `selector` everywhere instead of `div`.
You can remove those 2 vars and inline the string in the only function that use it, and in the test. It's not something that will change.